libfreefare/freefare.c

```diff
@@ -201,6 +205,17 @@ bool freefare_selected_tag_is_present(nfc_device *device)
     return (nfc_initiator_target_is_present(device, NULL) == NFC_SUCCESS);
 }
 
+/*
+ * Set NFC operation timeout
+ */
+void
+freefare_set_tag_timeout(FreefareTag tag, int timeout)
+{
+    if (tag) {
+        tag->timeout = timeout < 0 ? 0 : timeout;
```

Comment:
A negative timeout indeed does not make sense, but is allowed by `libnfc`, so we can simplify the setter to just `tag->timeout = timeout`.